case-study-azure-devops.md

```diff
@@ -0,0 +1,416 @@
+
+By using Azure DevOps and Microsoft team you and your team will have a really good way of collaborating and making sure that your project goes according to plan. You can show and adjust your team board in the teams app.
+
+<div style="width: 600px;">
+
+![Price Image](https://www.azuredevopslabs.com/labs/vstsextend/teams/images/kanbanboard_teams_new.png)
+
+<p style="padding: 0; margin: 0;">
+
+*Source: Azure DevOps Labs*
+
+</p>
+
+</div>
+
+*Reference [[10](#references)]*
+
+#### Azure DevOps Wiki
+
+Each project in Azure DevOps supports its own wiki, which enables you to conveniently write pages that help your team members and other users understand, use and contribute to your project.
+The wiki pages you create can be edited in your preferred IDE. You only need to clone the Wiki into your IDE and work on it there. It's powered by a git repository.
+
+Normally the developer will have a Markdown file to store documentation alongside the source code in a folder or just directly in the area you are working on. By using the Wiki feature it is possible to store your documentation and it's easy to add to your existing project. Wiki scans the files and make each markdown file into a wiki note.
+
+Here is a <a href="https://www.youtube.com/embed/Mmyw8rLs11c">Youtube video</a> showcasing the power of Azure DevOps Wiki features.
+
+*Reference [[11](#references)]*
+
+### Package Management
+
+Azure DevOps provides Package Management which is a feature that helps you manage and share packages across your organization. Package Management supports several popular package formats, including NuGet, npm, Maven, and Python packages.
+
+With Package Management you can publish packages to an internal or external feed and then consume them in your builds and releases. You can also manage packages from different package sources, such as public feeds like npmjs.com or Maven Central or a private feeds created by your team.
+
+When you publish a package, Azure DevOps stores the package and its metadata in the Package Management feed. You can then share the feed with others in the organization and use it as a source for your builds and releases.
+
+In your builds and releases, you can use the packages from the Package Management feed by specifying the feed in your build or release definition. Azure DevOps will then download the packages and make them available to your build agents or release agents.
+
+Package Management in Azure DevOps helps to centralize your package management and improve collaboration and reuse within the organization. The Package Management ensures that your builds and releases use consistent and up-to-date packages.
+
+*Reference [[12](#references)]*
+
+### Code Review and Feedback
+
+Azure DevOps offers a code review and feedback system similar to other popular platforms, like GitHub. You create a pull request and on that request you can add team members to look at the code and review it.
```

Comment:
A little more detail would be nice here, how does this fit with the name DevOps given the content of the lesson on DevOps in Module 3?

Reply:
Added some more content here.